packages/react/test/specs/utils/felaExpandCssShorthandsPlugin-test.ts

```diff
@@ -18,6 +18,21 @@ describe('felaExpandCssShorthandsPlugin', () => {
     })
   })
 
+  test('should expand handle "undefined" and "null"', () => {
```

Comment:
Can we add test for the `array` value scenario?

Reply:
Sure 🎸

Reply:
Added 👍